express-api/src/typeorm/Entities/NoteType.ts

```diff
@@ -0,0 +1,32 @@
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { BaseEntity } from './abstractEntities/BaseEntity';
+import { ProjectStatus } from './ProjectStatus';
+
+@Entity()
+export class NoteType extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  Id: number;
+
+  @Column({ type: 'character varying', length: 150 })
+  Name: string;
+
+  @Column('boolean', { default: false })
+  IsDisabled: boolean;
+
+  @Column('int', { default: 0 })
+  SortOrder: number;
+
+  @Column('text', { nullable: true })
+  Description: string;
+
+  @Column('boolean')
+  IsOptional: boolean;
+
+  // Status Relation
+  @Column({ name: 'status_id', type: 'int', nullable: true })
+  StatusId: number;
+
+  @ManyToOne(() => ProjectStatus, (ProjectStatus) => ProjectStatus.Id, { nullable: true })
+  @JoinColumn({ name: 'status_id' })
+  Status: ProjectStatus;
```

Comment:
Should we be more specific here? e.g. StatusId => ProjectStatusId

Reply:
Maybe... I wanted to keep it consistent with Task though.